Merge projects

Ben3094 opened the pull request:
[Modify] Namespace to be compliant with Net Standard API
[Add] FindClosestValue method to Range class
[Add] Derived Class (LinearDiscreteRange, DiscreteRange)

Hello,
I recently created a project similar to yours concerning range in C#. Before pushing my project on GitHub, I have found yours. Would you merge projects ?

Best regards

Ben3094 committed:
[Modify] Namespace to be compliant with Net Standard API
[Add] FindClosestValue methid to Range class
[Add] Derived Class (LinearDiscreteRange, DiscreteRange)

alansav commented:
That is great, before I merge the PR could you update the readme with the namespace and provide an example of the new functionality?

Ben3094 committed:
[Modify] Rename project to better match standard library names
[Add] Separate implementation of Range
[Modify] README now include new namespace name and new implementation
